services/ai-ml/gen-ai/features.yaml

```diff
@@ -130,3 +130,8 @@ features:
     description: |
       Ensure the service detects and filters abusive, harmful, and sensitive 
       information to ensure responsible and safe use of the service.
+  - id: CCC.GenAI.F26
+    title: Plugin Integrations
+    description: |
+        Ability for the model to use tools to complete a model interaction. For example web search, python code execution or external maths engine.
+      
```

Comment:
Hi @cronky, Thanks for the PR. It make sense to add it as a feature. Can you verify whether this feature exists in all 3 major CSPs? I did a quick lookup but couldn't find any evidence saying it is supported by Amazon Bedrock.

Reply:
This is more of a feature of the majority of foundation models (that are then offered as services by the CSPs) than just the CSPs native services - this came out of the working session in New York where it was agreed it was a gap.

For AWS they are referred to as tools here: https://aws.amazon.com/blogs/machine-learning/learn-how-to-build-and-deploy-tool-using-llm-agents-using-aws-sagemaker-jumpstart-foundation-models/

For an Azure example: https://www.agiliztech.com/2023/08/29/plugins-for-azure-openai-service-how-to-guide/#:~:text=Plugins%20are%20optional%20components%20you,and%20integrate%20with%20other%20services.

https://techcommunity.microsoft.com/t5/microsoft-azure-netherlands/how-to-leverage-plugins-for-azure-openai-service/m-p/3863986

Thanks

Oliver